v7/src/v7_encode_internal.h

```diff
@@ -92,7 +92,7 @@ void v7_encode_bp_canonical_block_buffer(v7_encode_state_t *enc, const bp_canoni
 
 int       v7_encoder_mpstream_write(void *arg, const void *ptr, size_t sz);
 int       v7_encoder_write_crc(v7_encode_state_t *enc);
-CborError v7_encoder_write_wrapper(void *arg, const void *ptr, size_t sz, CborEncoderAppendType at);
+int v7_encoder_write_wrapper(void *arg, const void *ptr, size_t sz);
```

Comment:
Just a general question for the PR, but how were these changes tested? How do you think QCBOR compared to TinyCBOR?